HW2/P5/driver.py

```diff
@@ -16,6 +16,22 @@
 def randcolor():
     return np.random.uniform(0.0, 0.89, (3,)) + 0.1
 
+# Grabbed from Morton ordering functions from 
+# https://en.wikipedia.org/wiki/Z-order_curve
+def less_msb(x, y):
+    return x < y and x < (x ^ y)
+
+def cmp_zorder(a, b):
+    j = 0
+    k = 0
+    x = 0
+    for k in xrange(2):
+        y = a[k] ^ b[k]
+        if less_msb(x, y):
+            j = k
+            x = y
+    return a[j] - b[j]
```

Comment:
z-order rocks. I developed one scheme myself but I think it is inferior to z-order